tests/testthat/test-process_repos.R

```diff
@@ -15,24 +15,6 @@ test_that("process_repos() returns results with repo metadata", {
   
   results |> is.list() |> expect_true()
   results |> names() |> expect_equal(c("GitStats", "DataFakeR"))
-  # TODO: check processed content
-})
-
-test_that("process_repos() returns results with repo metadata", {
   
-  my_project <- 
-    initialize_project("gitai_test_project") |>
-      set_github_repos(
-        repos = c("r-world-devs/GitStats", "openpharma/DataFakeR")
-      ) |>
-    add_files(file_types = "*.md") |>
-    set_llm() |>
-    set_prompt(system_prompt = "Summarize the user content if one sentence.") 
-      
-  my_project$gitstats$verbose_off()
-
-  my_project |> process_repos()
-
-  # TODO:
-
+  results |> purrr::map(~ nchar(.x$text) > 10) |> unlist() |> all() |> expect_true()
```

Comment:
Can you explain what we are testing here precisely?

Reply:
Right now, only that we have processed output from files content for each repo, and that outputted text is longer than a few characters. 
As we test it on live repos the output is not deterministic so we cannot take a snapshot of it.